### PR TITLE
Rename autocompleter participant card label to Search

### PR DESCRIPTION
**Pull request by sivaOSI**
[#138384237]

### CODE REVIEW

File: app/assets/javascripts/components/screenings/ScreeningEditPage.jsx [hunk 1]
```diff
@@ -77,14 +77,15 @@ export class ScreeningEditPage extends React.Component {
...
-            <span>Participants</span>
+            <span>Search</span>
           </div>
           <div className='card-body'>
             <div className='row'>
-              <div className='col-md-6'>
-                <label className='no-gap' htmlFor='screening_participants'>Participants</label>
+              <div className='col-md-12'>
+                <label className='no-gap pull-left' htmlFor='screening_participants'>Search for any person</label>
+                <i className='c-gray pull-left'>(Children, parents, collaterals, reporters, alleged perpetrators...)</i>
```
Comment (ThomasOSI) [inline app/assets/javascripts/components/screenings/ScreeningEditPage.jsx]: this &lt;i&gt; tag should be a &lt;span&gt;

File: spec/features/screening/participant/create_participant_spec.rb [hunk 2]
```diff
@@ -81,8 +81,8 @@ def build_participant_from_person_and_screening(person, screening)
 
     fill_in 'Title/Name of Screening', with: 'The Rocky Horror Picture Show'
 
-    within '#participants-card' do
-      fill_in_autocompleter 'Participants', with: 'Marge'
+    within '#search-card' do
```
Comment (ThomasOSI) [inline spec/features/screening/participant/create_participant_spec.rb]: This should probably be 
```within '#search-card', text: 'SEARCH' do```

File: spec/features/screening/participant/create_participant_spec.rb [hunk 3]
```diff
@@ -112,8 +112,8 @@ def build_participant_from_person_and_screening(person, screening)
 
   context 'searching for a person with the participant autocompleter' do
     scenario 'by first name' do
-      within '#participants-card' do
-        fill_in_autocompleter 'Participants', with: 'Marge'
+      within '#search-card' do
```
Comment (ThomasOSI) [inline spec/features/screening/participant/create_participant_spec.rb]: This should probably be 
```within '#search-card', text: 'SEARCH' do```

File: spec/features/screening/participant/create_participant_spec.rb [hunk 4]
```diff
@@ -134,8 +134,8 @@ def build_participant_from_person_and_screening(person, screening)
                     status: 200,
                     headers: { 'Content-Type' => 'application/json' })
 
-      within '#participants-card' do
-        fill_in_autocompleter 'Participants',
+      within '#search-card' do
```
Comment (ThomasOSI) [inline spec/features/screening/participant/create_participant_spec.rb]: This should probably be 
```within '#search-card', text: 'SEARCH' do```

File: spec/features/screening/participant/search_participant_spec.rb [hunk 5]
```diff
@@ -50,8 +50,8 @@
                     status: 200,
                     headers: { 'Content-Type' => 'application/json' })
 
-      within '#participants-card' do
-        fill_in_autocompleter 'Participants', with: 'Marge'
+      within '#search-card' do
```
Comment (ThomasOSI) [inline spec/features/screening/participant/search_participant_spec.rb]: This should probably be 
```within '#search-card', text: 'SEARCH' do```

File: spec/features/screening/participant/search_participant_spec.rb [hunk 6]
```diff
@@ -77,8 +77,9 @@
                     status: 200,
                     headers: { 'Content-Type' => 'application/json' })
 
-      within '#participants-card' do
-        fill_in_autocompleter 'Participants', with: person.ssn, result_should_contain: 'Marge'
+      within '#search-card' do
```
Comment (ThomasOSI) [inline spec/features/screening/participant/search_participant_spec.rb]: This should probably be 
```within '#search-card', text: 'SEARCH' do```

File: spec/features/screening/participant/search_participant_spec.rb [hunk 7]
```diff
@@ -102,8 +103,8 @@
                     status: 200,
                     headers: { 'Content-Type' => 'application/json' })
 
-      within '#participants-card' do
-        fill_in_autocompleter 'Participants', with: person.first_name
+      within '#search-card' do
```
Comment (ThomasOSI) [inline spec/features/screening/participant/search_participant_spec.rb]: This should probably be 
```within '#search-card', text: 'SEARCH' do```

File: spec/features/screening/participant/create_participant_spec.rb (hunk 2 above)
Comment (sivaOSI) [inline spec/features/screening/participant/create_participant_spec.rb]: ok Nate, I will change them..

File: app/assets/javascripts/components/screenings/ScreeningEditPage.jsx (hunk 1 above)
Comment (sivaOSI) [inline app/assets/javascripts/components/screenings/ScreeningEditPage.jsx]: Ok Nate,

**Pull request merged**